### PR TITLE
Update the README.

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This makes it easier for first time users by removing the apparent
need to install development things (since vagrant is now in many
disto's repos). Also remove the comment that it's our new server,
since it's been around for quite a while.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,35 +1,37 @@
 #Badger Vagrant
 
-Badger is the new [Student Robotics](http://studentrobotics.org) server. This
-is a vagrant configuration that will very easily go from scratch to a vagrantbox.
+Badger is the [Student Robotics](http://studentrobotics.org) server.
+This is a vagrant configuration that will very easily go from scratch
+to a VagrantBox.
```
Comment (fables-tales) [inline readme.md]: This is jargon. Imagine I'm new, what's a vagrant box?

File: readme.md [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
-   badger-vagrant`
-6. `vagrant up --provision`
-7. Wait about 20 minutes
-8. `vagrant ssh` to log into the box. You have passwordless sudo from the
+1. Install VirtualBox, usually in your distro's repos, or https://www.virtualbox.org/wiki/Downloads
+2. Install Vagrant, usually in your distro's repos, or http://www.vagrantup.com/downloads.html
+  * Or, if you like installing from source, you can install Ruby (1.9.3+,
+    including development headers: `yum install ruby-devel libffi-devel` or
+    `apt-get install ruby-dev libffi-dev`) and then `gem install vagrant`
+3. `git clone https://github.com/samphippen/badger-vagrant && cd badger-vagrant`
```
Comment (fables-tales) [inline readme.md]: I think all of these should have a `run` before them so that people know what they're doing.

**Comment by fables-tales** [comment]
A few notes. Otherwise great.

**Comment by PeterJCLaw** [comment]
This pull request is actually likely to include a bunch of other related fixes -- not just the README.

**Comment by fables-tales** [comment]
@PeterJCLaw how done is this should I merge it?

**Comment by PeterJCLaw** [comment]
I think it's done now, I've not got any other changes planned. Not sure the title of this PR is quite right though -- might want a tweak if you do merge it.

**Comment by fables-tales** [comment]
I'm just gonna merge it now.

**Pull request merged**